tests/mountingOptions/mocks.spec.ts

```diff
@@ -38,10 +38,10 @@ describe('mocks', () => {
         </div>
       `,
       computed: {
-        url() {
+        url(): string {
           return `/posts/${this.$route.params.id}`
         },
-        id() {
+        id(): string | string[] {
           return this.$route.params.id
```

Comment:
Concerned about this change, not 100% sure what's happening here

Reply:
`route.params[key]` is typed like this

![image](https://user-images.githubusercontent.com/19196536/169437520-19403f77-ce1c-470c-b32a-7588b7087b53.png)

What is your concern?

Reply:
The concern is that we shouldn't have to explicitly specify the type: the inference is somehow broken with V3.2.34

Reply:
Having to annotate the return type is something is there since V2, but this is not an issue with Vue test utils, is more a concern with Vue 3.2.34 release, all good on test utils IMO